File: app/src/main/java/com/zhxh/xjetpack/JetPackApp.java
package com.zhxh.xjetpack;

import android.app.Application;

import timber.log.Timber;

public class JetPackApp extends Application {
    @Override
    public void onCreate() {
        super.onCreate();

        //Plant a Timber Debug Tree here
        Timber.plant(new Timber.DebugTree());
    }
}

File: app/src/main/java/com/zhxh/xjetpack/JetPackApp.kt
package com.zhxh.xjetpack

import android.app.Application

import timber.log.Timber

class JetPackApp : Application() {
    override fun onCreate() {
        super.onCreate()

        //Plant a Timber Debug Tree here
        Timber.plant(Timber.DebugTree())
    }
}
